tedana/__init__.py

```diff
@@ -54,3 +45,5 @@
     path="tedana",
     cite_module=True,
 )
+
+del get_versions
```

Comment:
Why is this needed?

Reply:
It's not needed, but I think it's nice to have a clean set of locals for the package. I can remove it though.

Reply:
I see. We can leave it, just wanted to make sure I understood why it was there.